### ISSUE TITLE
Users are not able to vote on polls on Twitter

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#4670]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Issue reported by users on support saying they are unable to vote on polls posted on Twitter even with shields down.

> https://linkbubble.zendesk.com/agent/tickets/6107

**Expected behavior:**
Should be able to vote on polls posted on Twitter
- Platform (Win7, 8, 10? macOS? Linux distro?):
  Reported on OSX. Unable to repro on Windows 10
- Brave Version:
  0.12.4
- Steps to reproduce:
  1. Open any poll on Twitter
  2. User says they are unable to select an option to vote for the poll

Tried reproducing the issue on the same link that the user provided in the ticket and was able to do it with Shields up. 
- Screenshot if needed:
- Any related issues:

**Comment by neeklamy** [comment]
An easy way to find polls that are tagged: https://twitter.com/hashtag/poll

I know this worked perfectly well with previous versions of Brave. I’ve captured a video of me trying to vote for the next POTUS (_it’s about as much of a say I get, not being a US citizen_).

Choosing an option then clicking the vote button causes an animation, but not change in the vote count or my vote having registered:

![Brave Twitter vote failing](https://cloud.githubusercontent.com/assets/155358/19271782/03f67cc6-8fbe-11e6-98e2-d706c6182ae0.gif)

The console error is:

```
POST https://twitter.com/i/cards/api/v1.json?tweet_id=785309369512579072&card_na…4c26be5a3024faca1ccca8&forward=false&capi_uri=capi%3A%2F%2Fpassthrough%2F1

native_bundle_v1_9a9a3650e3ba83f9afdb31c69850edb3798609af.js:28
```

**Comment by bsclifton** [comment]
Reproduced- however, I was able to get past the issue by turning off fingerprint protection

**Comment by srirambv** [comment]
@bsclifton  Surprising I was able to do get past and vote without making any changes to fingerprint protection. The only thing that was disabled on shields was the block script.

**Comment by neeklamy** [comment]
As a follow up, I can successfully vote on some polls but not others. Sometimes the vote goes through from the https://twitter.com/hashtag/poll page, other times I have to open a tweet (click on the header to expand it) for the vote to count.

The error in the console remains much the same:

```
Failed to load resource: the server responded with a status of 403 ()

https://twitter.com/i/cards/api/v1.json?tweet_id=786107584319225856&card_na…0a42d7a3b71f74de3edf90f&forward=true&capi_uri=capi%3A%2F%2Fpassthrough%2F1
```

**Comment by psmalone** [comment]
Confirming still at least an intermittent problem on 0.12.9 on El Capitan. I can't vote on some polls even with shields down and refreshing, and no fingerprinting protection.

**Comment by DivineOmega** [comment]
I'm also able to reproduce this problem (Brave 0.12.9, Ubuntu 16.04 LTS). Voting causes a flickering animation, but the vote count does not seeem to be affected.

This may also be affecting the ability to create polls on Twitter, as I receive the same console error as @neeklamy when attempting to do this.

**Comment by pixelass** [comment]
As mentioned in #5748 I cannot create polls using brave. Not sure if this is related. Needs further investigation though.

**Comment by bsclifton** [comment]
+1 from user @jaydson https://github.com/brave/browser-laptop/issues/4670

**Comment by jonathansampson** [comment]
I was able to create a poll, and vote with preview build 7 of 0.13.0.

**Comment by Liunkae** [comment]
Closing as fixed in 0.13.0.
